components/nav/Nav.tsx

```diff
@@ -2,16 +2,29 @@ import Image from 'next/image';
 import Logo from '@/components/assets/logo.svg';
 import ProfileDropdown from '@/components/nav/ProfileDropdown';
 import Link from 'next/link';
+import { headers } from 'next/headers';
+
+export default async function NavBar() {
+  const headersList = headers();
+  const referer = headersList.get('referer');
+  let url = '/private-cloud/products/all';
+  if (typeof referer === 'string') {
+    if (referer.split('/')[3] === 'public-cloud') {
```

Comment:
let's create a variable on top to reuse the variable for two conditions.

Reply:
currentURL variable is created